timewarp_gl/utils/hmd.hpp

```diff
@@ -49,7 +50,7 @@ class HMD {
     static void  GetDefaultHmdInfo(const int displayPixelsWide, const int displayPixelsHigh, const float displayMetersWide,
                                    const float displayMetersHigh, const float lensSeparation, const float metersPerTanAngle,
                                    const float aberration[4], hmd_info_t& hmd_info);
-    static void  BuildDistortionMeshes(mesh_coord2d_t* distort_coords[NUM_EYES][NUM_COLOR_CHANNELS], hmd_info_t& hmdInfo);
+    static void  BuildDistortionMeshes(std::array<std::array<mesh_coord2d_t*, NUM_COLOR_CHANNELS>, NUM_EYES> distort_coords, hmd_info_t& hmdInfo);
```

Comment:
Oh, so this is a 2d array of pointers?

Is there any way to do this without the pointer indirection?

No big if that's a lot of work.